web_src/js/index.js

```diff
@@ -755,6 +755,17 @@ async function initRepository() {
     });
   }
 
+  // Repo Creation
+  if ($('.repository.new.repo').length > 0) {
+    $('input[name="gitignores"], input[name="license"]').on('change', () => {
+      const gitignores = $('input[name="gitignores"]').prop('checked');
+      const license = $('input[name="license"]').prop('checked');
```

Comment:
I just realized that these are not checkboxes :D These lines should have been:
```js
      const gitignores = $('input[name="gitignores"]').val() !== '';
      const license = $('input[name="license"]').val() !== '';
```

Reply:
Right I missed it too. `.val()` should be sufficient and possible more future-proof just in case the value of a empty select box ever changes to `null` or `undefined`.

Reply:
https://github.com/go-gitea/gitea/pull/12299

Reply:
Thanks for PR. I don't think jQuery would return null or undefined but not sure. Have not used jQuery for quite some time already 😂

Reply:
Nothing to do with jQuery, but I guess DOM `<input>` value is probably guaranteed to be a string (even for `type=number` 😆).